cli/transmission-cli.1

```diff
@@ -32,7 +32,7 @@
 .Op Fl h
 .Op Fl m | M
 .Op Fl p Ar port
-.Op Fl t Ar tos
+.Op Fl -dscp Ar dscp
```

Comment:
```suggestion
.Op Fl t Ar tos
```
Should be reverted to original tos option.

Reply:
Good catch. Fixed in 8dfaa6a8dc204b7c3a29a314fb488b5ff866aa77